wasmtime-environ/src/compilation.rs

```diff
@@ -9,7 +9,7 @@ use cranelift_wasm::{DefinedFuncIndex, FuncIndex, WasmError};
 use std::ops::Range;
 use std::vec::Vec;
 
-type Functions = PrimaryMap<DefinedFuncIndex, Vec<u8>>;
+type Functions = PrimaryMap<DefinedFuncIndex, (Vec<u8>, ir::JumpTableOffsets)>;
```

Comment:
The tuple `(Vec<u8>, ir::JumpTableOffsets)` appears several times, and there are a few `.0`'s in the code. What would you think of making this a struct, maybe named `CodeAndJTOffsets` or so?